### ISSUE TITLE
Incorrect Disk Write Bytes

### ISSUE THREAD

**Issue by SeriousHoax** [winsiderss/systeminformer#1980]
### Brief description of your issue

System Informer is showing incorrect disk write bytes for ESET Antivirus's "ekrn.exe" process. The disk write bytes shown by System Informer are significantly higher than the actual number. Microsoft's Process Explorer shows the correct numbers. 
This must be more of a recent bug because when I checked a few months ago everything was fine.
It seems some of the I/O writes not all of which are disk writes are being counted as disk writes for the process. 
As you can see from the screenshot, the I/O writes for both are the same but there's a massive difference in disk write bytes.
728 KB vs 866 MB. 
![sys](https://github.com/winsiderss/systeminformer/assets/29574622/e111c59c-93bc-4357-b0c6-9691b81f07cc)

### Steps to reproduce (optional)

Install any of ESET's antivirus products. In my case, it's ESET Smart Security Premium. After that monitor its disk write bytes when it updates or performs what they call, a "Startup scan" which is performed every time the product updates its signature and at system startup. 

### Expected behavior (optional)

Show accurate value. 

### Actual behavior (optional)

The value shown is much higher than the actual value.

### Environment (optional)

```shell
System Informer Nightly 3.0.7459
Windows 11 23H2
```

**Comment by ge0rdi** [comment]
In my case the `Disk write bytes` number is even higher than `I/O write bytes`:
![image](https://github.com/winsiderss/systeminformer/assets/2453094/fc9f33c5-d043-4bb7-be11-66087260525d)
That doesn't make much sense as Disk I/O should be subset of total I/O.

Note that `Disk I/O` statistics are provided by `ExtendedTools` plugin.
Maybe something is wrong there?

**Comment by SeriousHoax** [comment]
> In my case the `Disk write bytes` number is even higher than `I/O write bytes`: ![image](https://private-user-images.githubusercontent.com/2453094/304764758-fc9f33c5-d043-4bb7-be11-66087260525d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc5MjQ0ODYsIm5iZiI6MTcwNzkyNDE4NiwicGF0aCI6Ii8yNDUzMDk0LzMwNDc2NDc1OC1mYzlmMzNjNS1kMDQzLTRiYjctYmUxMS02NjA4NzI2MDUyNWQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDIxNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAyMTRUMTUyMzA2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MWE1ZmZkMjhlYjcyOGY5YzAxMTBlN2U4MjFjZGMwN2U0YjU5ZjgwNzEyNTA0NmJjNGZjOWExNjQ1NTU3MGQ3NiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.GZIxjPZdl1EbHW5zB1rULXX42PXZWKYWaVAyQApGF94) That doesn't make much sense as Disk I/O should be subset of total I/O.
> 
> Note that `Disk I/O` statistics are provided by `ExtendedTools` plugin. Maybe something is wrong there?

You're right. Same for me.
![image](https://github.com/winsiderss/systeminformer/assets/29574622/47b7f265-0252-45c3-81dc-066a202ec9cf)

**Comment by dmex** [comment]
Are either of you using DevDrive on Win11?

**Comment by SeriousHoax** [comment]
> Are either of you using DevDrive on Win11?

No, I'm not using it.

**Comment by ge0rdi** [comment]
> Are either of you using DevDrive on Win11?

Nope.

**Comment by ge0rdi** [comment]
BTW It is not related just to `ekrn.exe` counters.
Many other processes have bigger `Disk write bytes` than `I/O write bytes`.

On Win10 too (though here seems to be the discrepancy somehow smaller?).

**Comment by SeriousHoax** [comment]
> BTW It is not related just to `ekrn.exe` counters. Many other processes have bigger `Disk write bytes` than `I/O write bytes`.
> 
> On Win10 too (though here seems to be the discrepancy somehow smaller?).

Yeah, after you mentioned that's what I'm noticing also. Hope you guys find the reason and the fix soon.

**Comment by dmex** [comment]
Windows 10 and 11 include new disk counters for individual threads and processes within the same ethread/eprocess objects for the entire lifetime of the process. The previous disk counters were only contained within separate ETW events which reset whenever you restart the application.

> Process Explorer shows the correct numbers.

Process Explorer doesn't support the new disk counters and resets whenever you close the program.

**Comment by ge0rdi** [comment]
@dmex 
What about the discrepancy between Disk I/O and Total I/O?
Shouldn't total I/O be always higher (or equal) than disk I/O?

**Comment by SeriousHoax** [comment]
To be clear, I know that Process Explorer needs to be running and it resets after closing the app. In my initial screenshot, Process Explorer was run as admin before I took the measurement and was not closed. So that was not the issue. More clarification on which value represents the actual amount of data being written on the disk would be helpful. 728 KB or 866 MB? Also what @ge0rdi asked above about disk writes being lower than I/O writes.

**Comment by dmex** [comment]
It's the total accumulate value since startup. If you open a 100mb file with notepad it'll remain 100mb until the process terminates and these statistics are maintained by the kernel independently of anything System Informer does.
